src/Console/Command/Migrate.php

```diff
@@ -37,8 +43,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         
         // environment
         $environment = $input->getOption('environment');
+        $configPath = $input->getOption('configuration');
```

Comment:
This is path to die with configs, not configuration

Reply:
Okay, can you suggest any other option? I can name it path

Reply:
`configuration` is good but it must be fully qualified filename:

```
phpunit -c myOwnPhpunitConfig.xml
```

Reply:
and it must be defined globally for all app, not just for migrate command

Reply:
but how I can keep state of migrator if migrator's config might be placed anywhere?

Reply:
@sokil In my case it is better not to keep a state but pass when I need it. Like phinx

```
#!/bin/bash

cd "$( dirname "$( dirname "${BASH_SOURCE[0]}" )" )"

./vendor/bin/phinx --configuration=./app/db/phinx.php migrate $@
```

Reply:
What state you talk about? I told you about config instance, read in any command. This Getter must be idempotent, without passed arguments and return same value every time you call it.